fdbserver/storageserver.actor.cpp

```diff
@@ -3360,6 +3361,40 @@ bool storageServerTerminated(StorageServer& self, IKeyValueStore* persistentData
 		return false;
 }
 
+ACTOR Future<bool> memoryStoreRecover(IKeyValueStore* store, Database cx, UID id)
+{
+    if(store->getType() != KeyValueStoreType::MEMORY) {
+        return false;
+    }
+
+    state Transaction tr( cx );
+	state int noCanRemoveCount = 0;
+	loop {
+		try {
+			if (noCanRemoveCount >= 5) {
+			// retry five times, can't remove storage server, needs to wait recovery done
```

Comment:
Doesn't this mean that you will give up after 5 seconds? What is the rational behind this? I would expect that you want to poll until the storage node is fully recovered?